DESCRIPTION

```diff
@@ -48,6 +48,7 @@ Suggests:
     tergm (>= 4.0.0),
     ergm.count (>= 4.0),
     ergm.userterms (>= 3.10.0),
+    ergm.tapered (>= 1.0.0),
```

Comment:
Since ergm.tapered is not on CRAN, we also need to add a Remotes: directive.

That said, we might want to keep it away from requirements to avoid a chicken-and-egg problem the first time we release.